src/player.cpp

```diff
@@ -1598,6 +1597,19 @@ void player::recalc_speed_bonus()
 
 int player::run_cost(int base_cost, bool diag) const
 {
+    return move_cost_encumbrance( base_cost, diag );
```

Comment:
This rename is weird and misleading: `move_cost_encumbrance` sounds like extra move cost due to encumbrance, not the entire run cost.

`move_cost_encumbrance` should be just the part at the very bottom of the function. You can split the function into `move_cost_base` and `move_cost_encumbrance` if you need to exclude body parts (with `move_cost_base` not having body part exclusion arguments).